<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>976</width>
    <height>542</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>976</width>
    <height>496</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="studentsList">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>50</y>
      <width>211</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>学生列表</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addStudent">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>160</y>
      <width>211</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>增加学生</string>
    </property>
   </widget>
   <widget class="QPushButton" name="deleteStudent">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>160</y>
      <width>211</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>删除学生</string>
    </property>
   </widget>
   <widget class="QPushButton" name="searchStu">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>90</y>
      <width>211</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>查询学生</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addStuClass">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>280</y>
      <width>211</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>增加学生选课</string>
    </property>
   </widget>
   <widget class="QPushButton" name="deleteStuClass">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>280</y>
      <width>211</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>删除学生选课</string>
    </property>
   </widget>
   <widget class="QPushButton" name="searchClass">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>200</y>
      <width>211</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>查询课程</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonExit">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>400</y>
      <width>251</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>退出Exit</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>10</y>
      <width>401</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>欢迎来到学生选课信息管理系统！</string>
    </property>
   </widget>
   <widget class="QPushButton" name="classesList">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>50</y>
      <width>211</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>课程列表</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>976</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
